src/main/groovy/com/getkeepsafe/dexcount/DexFile.groovy

```diff
@@ -80,16 +80,24 @@ class DexFile {
             }
         }
         // convert it to DEX format by using the Android dx tool
-        def androidSdkHome = System.getenv("ANDROID_HOME")
+        def androidSdkHome = DexMethodCountPlugin.sdkLocation
         if (androidSdkHome == null) {
             throw new Exception("ANDROID_HOME env variable not defined!")
```

Comment:
This exception is no longer correct 100% of the time. This check would also be best suited in `DexMethodCountPlugin` when you first initialize the location.